deploy/kubernetes/manifests/node.yaml

```diff
@@ -0,0 +1,79 @@
+#TODO: Force DaemonSet to not run on master.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: gcp-filestore-csi-node
+  namespace: gcp-filestore-csi-driver
+spec:
+  selector:
+    matchLabels:
+      app: gcp-filestore-csi-driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: gcp-filestore-csi-driver
+    spec:
+      serviceAccount: gcp-filestore-csi-node-sa
+      hostNetwork: true
+      containers:
+        - name: csi-driver-registrar
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/driver-registrar:v0.3.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi
+          #  - name: registrar-socket-dir
+          #    mountPath: /var/lib/csi/sockets/
+        - name: gcp-filestore-driver
+          securityContext:
+            privileged: true
+          imagePullPolicy: Always
+          image: gcr.io/google-containers/volume-csi/gcp-filestore-csi-driver:v0.1.0
+          args:
+            - "--v=5"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--nodeid=$(KUBE_NODE_NAME)"
+            - "--node=true"
+          env:
+            - name: CSI_ENDPOINT
+              value: unix:/csi/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: kubelet-dir
+              mountPath: /var/lib/kubelet
+              mountPropagation: "Bidirectional"
+            - name: plugin-dir
+              mountPath: /csi
+        - name: nfs-services
```

Comment:
? can this be done in the same container based on `--node=true` maybe?

Reply:
I wanted to keep it separate because:
1) it launches a bunch of other processes
2) it can have its own health monitoring/probes
3) it's not related to csi, it's a system dependency